mono/utils/mono-threads-coop.c

```diff
@@ -640,6 +639,36 @@ mono_threads_suspend_policy_name (void)
 	}
 }
 
+static gboolean
+blocking_transition_getenv_compat (void)
+{
+	static int inenv = -1;
+	if (G_UNLIKELY (inenv == -1))
+		inenv = g_hasenv ("MONO_ENABLE_BLOCKING_TRANSITION");
```

Comment:
We should remove support for this env variable, it's not really useful and not even documented.